Tiltfile

```diff
@@ -293,13 +310,48 @@ def enable_provider(name, debug):
             links = links,
         )
 
-def find_manager(yaml):
-    manifests = decode_yaml_stream(yaml)
+        provider_yaml, object_name = get_provider(manifests)
+        k8s_yaml(blob(provider_yaml))
+        k8s_resource(
+            objects = [object_name],
+            resource_deps = ["provider_crd"],
+            new_name = label.lower() + "_cr",
+            labels = [label],
+        )
```

Comment:
Let's drop this, so we keep our groups focused only what actually changes during dev (binary/controllers); provider_crd will go into uncategorized, but I think it is fine

Reply:
I assume you mean the `labels` field. I have to define the resource to be able to add the dependency

Reply:
What if we "append provider" yaml to the component yaml for each provider, and we create a dependency there?

Reply:
Done. But I still need the object name. `workload` only identifies the workload and if I don't add the provider object to the k8s_resource it's still deployed as part of unlabeled/uncategorized